apps/src/templates/PauseButton.jsx

```diff
@@ -60,27 +68,27 @@ class PauseButton extends React.Component {
 const styles = {
   icon: {
     lineHeight: 'inherit',
-    color: color.white
+    fontSize: '2.8em'
```

Comment:
Out of curiosity, are we using `em` sizes for elements in this area?

Reply:
Good call. We were using `px` elsewhere, so I updated here.